splink/comparison_level_library.py

```diff
@@ -139,6 +139,34 @@ def create_label_for_charts(self) -> str:
         return f"Levenshtein distance of {self.col_name} <= {self.distance_threshold}"
 
 
+class DamerauLevenshteinLevel(ComparisonLevelCreator):
+    def __init__(self, col_name: str, distance_threshold: int):
+        """A comparison level using a Damerau-Levenshtein distance function
+
+        e.g. damerau_levenshtein(val_l, val_r) <= distance_threshold
+
+        Args:
+            col_name (str): Input column name
+            distance_threshold (int): The threshold to use to assess
+                similarity
+        """
+        super().__init__(col_name)
+        self.distance_threshold = distance_threshold
+
+    def create_sql(self, sql_dialect: SplinkDialect) -> str:
+        col = self.input_column(sql_dialect)
```

Comment:
We might as well be consistent here:
```suggestion
        col_l, col_r = self.input_column(sql_dialect).names_l_r
```